README.md

```diff
@@ -114,3 +114,136 @@ spec:
+export MODULE_REGISTRY=registry.localhost:5001/unsigned
+```
+
+4. Build Keda module
+```bash
+make module-build
+```
+
+This will build oci image for keda module and push it to the registry and path as defined in `MODULE_REGISTRY`.
+
+5. Build Keda manager image
+```bash
+make module-image
+```
+
+This will build docker image for keda manager and push it to the registry and path as defined in `IMG_REGISTRY`.
+
+6. Verify if the module and the manager's image are pushed to the local registry
+
+```bash
+curl registry.localhost:5001/v2/_catalog
+{"repositories":["unsigned/component-descriptors/kyma.project.io/module/keda","unsigned/operator-images/keda-operator"]}
+```
+
+7. Inpect the generated module template
+
+The following are temporary workarounds. Probabaly will be mitigated in the future.
+
+Edit the `template.yaml` file and:
+
+ - change the `target` to `control-plane`
+This is only required in the single cluster mode
+
+```yaml
+spec:
+  target: control-plane
+```
+
+- change the existing repository context in spec.descriptor.component:
+(Because pods inside the k3d cluster use the docker-internal port of the registry, it will try to resolve the registry against port 5000 instead of 5001. K3d has registry aliases but module-manager is not part of k3s and thus does not know how to properly alias `registry.localhost:5001`)
+
+```yaml
+repositoryContexts:                                                                           
+- baseUrl: registry.localhost:5000/unsigned                                                   
+  componentNameMapping: urlPath                                                               
+  type: ociRegistry
+```
+
+8. Install modular kyma on the k3d cluster
+
+This will install the latest versions of `module-manager` and `lifecycle-manager`
+
+You can  use --template option to deploy the keda module manifest from the beginning or apply it via kubectl later.
+
+```bash
+kyma alpha deploy  --template=./template.yaml
+
+- Kustomize ready
+- Lifecycle Manager deployed
+- Module Manager deployed
+- Modules deployed
+- Kyma CR deployed
+- Kyma deployed successfully!
+
+Kyma is installed in version:
+Kyma installation took:		18 seconds
+
+Happy Kyma-ing! :)
+```
+
+Kyma installation is ready, but no module is activated yet
+```bash
+kubectl get kymas.operator.kyma-project.io -A
+NAMESPACE    NAME           STATE   AGE
+kcp-system   default-kyma   Ready   71s
+```
+
+Keda Module is a known module, but not activated
+```bash
+kubectl get moduletemplates.operator.kyma-project.io -A 
+NAMESPACE    NAME                  AGE
+kcp-system   moduletemplate-keda   2m24s
+```
+
+9. Give module manager permission to install CRD cluster wide
+
+This is temporary workaround for the local mode only:
+(As module-manager  does not have the permissions in local mode for creating CRDs; since in local-mode it uses service account, while in remote mode, an administrative kubeconfig is expected)
```

Comment:
Can you rework this sentence a little bit? I'm not sure if I understand it correctly.